### DISCUSSION TITLE
Decode the hidden quote 🤐

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1266] [Galaxy Trivia] [answered]
This is a type of a shift cipher in which the message is encrypted by replacing the original letters with “x” number of characters ahead in the alphabet. The offset is described by the ‘salt’. So, for example, if the first digit of the salt is 4, the first original character will be 4 characters ahead of the encrypted letter. Then, the second character will follow the offset given by the second digit of the salt. Start over from the first digit when the salt is exhausted and continue until the original letters of the hidden message are revealed. You have finished the puzzle when you have decoded the full message.

```
DZ JS YMGOCZM OVU
HY TUHFGIA LPAM
RCY QBWHCN NEYU
RMPFA MWT GO QXM
XZAVOPY E XMPFAH'P
NUDG.

salt: 5636452
```

**Bonus:** Name the author.

**Comment by tomerf** [comment, marked as answer]
"IF MY CRITICS SAW ME WALKING OVER THE THAMES THEY WOULD SAY IT WAS BECAUSE I COULDN'T SWIM."
If was said by Margaret Thatcher.

**Reply by ParanoidUser** [reply]
Good job! 👏👏
